### DISCUSSION TITLE
The co-pilot of Github is not working. He responded that I should log in to github, but I was already logged in

### DISCUSSION THREAD

**Discussion by vktrbr** [community/community#37675] [Copilot Conversations] [answered]
Hello! 👋

My copilot version: 1.1.34.2009
And my code complections does not working. But I was already logged in. 

<img width="520" alt="Снимок экрана 2022-10-29 в 00 40 22" src="https://user-images.githubusercontent.com/52676181/198737456-a2023eae-5a73-4f8b-80dc-1f519b9a8182.png">

Please help me solve this problem.

**Comment by rpriyanshu9** [comment, marked as answer]
Try logging out and logging in from your IDE. And check if your copilot license is still running or not.

**Reply by vktrbr** [reply]
thanks a million ! ❤️‍🔥

**Comment by Moses-Code-Dev** [comment]
What is your Os version?

**Reply by vktrbr** [reply]
macos monterey 12.6 (21G115)
